vendor/github.com/hashicorp/nomad/api/api.go

```diff
@@ -943,7 +943,10 @@ func decodeBody(resp *http.Response, out interface{}) error {
 	}
 }
 
-// encodeBody is used to encode a request body
+// encodeBody prepares the reader to serve as the request body.
```

Comment:
wait, are Nomad vendoring itself?

Reply:
i'm going to let @shoenig answer this

Reply:
Yes, it's more or less a mandate [from Go 1.14](https://golang.org/doc/go1.14#go-command)
> When the main module contains a top-level vendor directory and its go.mod file specifies go 1.14 or higher, the go command now defaults to -mod=vendor for operations that accept that flag.

Without setting `-mod=mod`, the `vendor` directory must be in the expected state, including vendoring of any imported submodules.

There is an [open issue](https://github.com/golang/go/issues/33789) to improve the ergonomics around this use case. We're also working behind the scenes on the feasibility of removing the `vendor` directory entirely.